Makefile

```diff
@@ -106,6 +106,9 @@ test:
 test-e2e:
 	bats -t ${BATS_TESTS_FILE}
 
+test-gator: gator
```

Comment:
```suggestion
test-gator: bin/gator-linux-amd64
```

Reply:
`gator` is a makefile target that already does this:

```
.PHONY: gator
gator: bin/gator-$(GOOS)-$(GOARCH)
	mv bin/gator-$(GOOS)-$(GOARCH) bin/gator
```

Reply:
it's not getting the correct values, but just a cosmetical change. https://github.com/open-policy-agent/gatekeeper/runs/4472699630?check_suite_focus=true#step:4:10

Reply:
Done!

Reply:
It doesn't work now. I don't understand Makefile magic enough to know why your suggestion makes the test fail

Reply:
It works locally, but it makes it fail in GitHub actions.

Reply:
Ok, I've got it working. This doesn't move the binary, so it just executes `bin/gator--`.